hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -266,7 +266,7 @@ public class RSRpcServices extends HBaseRpcServicesBase<HRegionServer>
   /**
    * Default value of {@link RSRpcServices#REGION_SERVER_RPC_MINIMUM_SCAN_TIME_LIMIT_DELTA}
    */
-  private static final long DEFAULT_REGION_SERVER_RPC_MINIMUM_SCAN_TIME_LIMIT_DELTA = 10;
+  static final long DEFAULT_REGION_SERVER_RPC_MINIMUM_SCAN_TIME_LIMIT_DELTA = 10;
```

Comment:
Is the configuration variable associated with this constant ever changed? It is     "hbase.region.server.rpc.minimum.scan.time.limit.delta" defined in RsRpcServices. @bbeaudreault @Apache9 
Do we need it? Seems like one of those knobs that nobody will ever turn.

Reply:
I agree. Only thing i could think of is maybe for like an s3-backed cluster where you know latency will be higher? 10ms may be too small and still timeout before starting. But that's really grasping.

I'm happy to remove if that's what we want. @Apache9